LondonBritishLibrary/orient/BLorient674.xml

```diff
@@ -0,0 +1,418 @@
+                                        <q xml:lang="am"/>
+                                    </item>
+                                    
+                                    <item xml:id="a11">
+                                        <locus target="#185vb"/>
+                                        <desc type="ReceiptNote">Amharic receipt note or
+                                            inventory of liturgical items and books, mentioning
+                                            <persName ref="PRS13703Asme">Yabǝčču Daggʷāš ʾAṣme</persName>, dated to the reign of ʾIyoʾas,
+                                            probably to be identified with <persName ref="PRS5636IyoasI"/>.</desc>
+                                        <q xml:lang="am"/>
+                                    </item>
+                                    
+                                    <item xml:id="a12">
+                                        <locus target="#185vc"/>
+                                        <desc type="ReceiptNote">Amharic receipt note or
+                                            inventory of liturgical items and books, mentioning
+                                            <persName ref="PRS13703Asme">Daggwāš ʾAṣme</persName> 
+                                            and <persName ref="PRS13704Isayeyyas">Qes Gabaz ʾIsāyǝyyās</persName>, dated to the reign of ʾIyāsu,
+                                            probably to be identified with <persName ref="PRS5616IyasuI"/> or <persName ref="PRS5617IyasuII"/></desc>
+                                        <q xml:lang="am"/>
+                                    </item>
+                                    
+                                    <item xml:id="a13">
+                                        <locus target="#191rb"/>
+                                        <desc type="ReceiptNote">Amharic receipt note or
+                                            inventory of liturgical items and books, mentioning
+                                            <persName ref="PRS13712AdaraG">ʾAdarā Giyorgis</persName>.</desc>
+                                        <q xml:lang="am"/>
+                                    </item>
+                                    
+                                    <item xml:id="a14">
+                                        <locus target="#191rc"/>
+                                        <desc type="ReceiptNote">Amharic receipt note or
+                                            inventory of liturgical items and books, mentioning
+                                            <persName ref="PRS13694Qale">Qes Qāle</persName>, dated to the reign of <persName ref="PRS2377Bakaffa"/>.</desc>
+                                        <q xml:lang="am"/>
+                                    </item>
+                                    
+                                    <item xml:id="a15">
+                                        <locus target="#191vb"/>
```

Comment:
"ReceiptNote" has a limited circulation in Bm, would "Inventory" not be better?

Reply:
I was hesitating, because the nature of these notes in unclear to me. They mostly contain the verb ተመጠወ